.travis.yml

```diff
@@ -79,17 +79,32 @@ before_deploy:
 
 # On success, upload a package to an S3 Bucket. Used for continuous deployment.
 deploy:
-  access_key_id: "AKIAJMZN4F5L5KXPQKVQ"
-  secret_access_key:
-    secure: owxrrQc4i1jfAwvASFewdMzNdi93zpwnoEhsTJQS/f3SkD083XpefUr4H7Mg8cZiq5grT6NpoYkG6QDoxz30Kn5IbDIHewRy3I11uwz02JGUkzxv+/7hLOnoM/LM993WWXA+YQJmrO4HvncDy3Hgbw98xkKOeqsQ1XlYR8Lp5NUQnwJtLkouOJYfJSU/ZuIyetYC3bO2JkF/smtEPmVPJ8ZJjAQEqCdiUJsvMWKDlDh+cLOKM6EBTorE8GTECfMY+HUy74nH7xdkJ2xQm6PjO2wJVtcgCW377Jfn7IJqMhmlhA6fnkguyb58S5NkwwSynFizm7GidtbAxTeaIHimB3B6ImWUQu/maiWrWpMZGyqoC+l3+zQ/jg5Pam9Crtff8odFNT/tvwVGMTxuVkUxYRZ87XGM+G4D29xV/FGs0oqyn3Jw4SPXmg1bBymJOwmL+YogYFvJAR77MmbBUcf6SCogJzvI4Z2IFe9afMCbZaNeSD1GGh+EYCjJIKrpc9sY2ZxON1Mgv1bQ9FnSrDxyCUYEsshh8x9AaAFcYdGRGc6V0IT/5gopH3gf+XC+Q2OQHGC5oiaJ0+R4BoT3YuxEYfpBHvrpWk30Ug8C9ZgxoGK2xjabKiJ/2LlKe+xHoe77ZWZCkc1RocZnSHFBaFgaQrzppWsty04WGHl5fIVgH1Q=
-  provider: s3
-  bucket: "datacube-core-deployment"
-  region: "ap-southeast-2"
-  local_dir: dist
-  skip_cleanup: true
-  on:
-    branch: develop
-    python: "3.6"
+  - provider: s3
```

Comment:
It'd be great to add some comments about what this is achieving and why.

Also, I'm not sure if it's possible, but maybe these sections could be merged? Not worth spending too long on it though.